src/form-question-types.rst

```diff
@@ -1314,29 +1314,42 @@ type
 appearance
   *none*
 
-Captures the current geolocation from the device.
+Captures the current geolocation from the device. The location is displayed in degrees-minutes-seconds (DMS) notation and is stored in `decimal degrees <https://en.wikipedia.org/wiki/Decimal_degrees>`_ with altitude and accuracy. Learn more about the format of resulting data in :ref:`the location widgets section <location-widgets>`.
```

Comment:
resulting format url isn't helpful. Maybe you meant http://getodk.github.io/xforms-spec/#data-types